apps/admin-page-e2e/src/integration/components/layout.spec.ts

```diff
@@ -0,0 +1,142 @@
+describe('LayoutComponent', () => {
```

Comment:
There should be a test to validate that the sidebar can be collapsed
While you can't put `data-cy` attribute on the button for collapsing the sidebar, you can get `nz-sider-trigger` element and then click on the `nz-icon` within it